core/cap-0034.md

```diff
@@ -570,8 +608,8 @@ nominate, in new protocols only (it preserves its old behavior in old
 protocols), changes in the following ways:
 
   - It chooses a "combined" `closeTime` directly from the `StellarValue`
-  containing the winning transaction set.  In particular, it does not need
-  any trimming of the winning transaction set.  (That set should already be
+  containing the combined transaction set.  In particular, it does not need
```

Comment:
we're not combining transaction sets though, right? In the rest of the proposal you're using "chosen" or "selected", so I'd just use either one here as well to avoid confusion.

Reply:
Done.  I also replaced (with "composited") a few other uses of "combined" that might have sounded like they were excluding the possibility of simply selecting one particular transaction set (as we do).